### ISSUE TITLE
Installing via AUR provided a broken package.

### ISSUE THREAD

**Issue by dariusity** [leftwm/leftwm#172]
I compiled leftwm-git via AUR and upon success I installed and tried running it but it received some errors related to RUST_BACKTRACE environment not set.
It also shows display_server not available.
There are only these files that were installed by the package:
/usr/bin/leftwm
/usr/bin/leftwm-state
/usr/bin/leftwm-worker
/usr/share/leftwm/themes/
/usr/share/xsessions/leftwm.desktop

Is there something missing?

**Comment by lex148** [comment]
No, LeftWM is very light weight. That looks like all that should be installed.
Would you be willing to post errors. I would love to help you get this fixed ASAP.

**Comment by dariusity** [comment]
That's one of the reasons I wanted to use leftwm. I'll revert back to you with the errors asap. Are you Arch user? Just wondering if the package was done properly as it used to have an error few months back. Cheers.

**Comment by lex148** [comment]
I'm on Ubuntu for work and Arch for home, both running LeftWM. You shouldn't have to touch your .xinitrc file unless you want to. 
If you booted to the black screen it sounds like you are there, but something is not quite right with your theme. 
LeftWM doesn't care what is in your theme folder except it wants and `up` and a `down` script. 

I would recommend doing the following: 
- [Download a copy LeftWM](https://github.com/leftwm/leftwm/archive/master.zip)
- copy the contents basic_polybar to ~/.config/leftwm/themes/basic_polybar
- double check that `~/.config/leftwm/themes/up` is executable. 
- link basic_polybar to be your active theme 
```
ln -s ~/.config/leftwm/themes/basic_polybar ~/.config/leftwm/themes/current
```

Here is what my theme folder looks like for reference: 
```
.config/leftwm/themes
❯ ls -alh
total 16K
drwx------ 4 lex lex 4.0K Jan 29 09:53 ./
drwx------ 3 lex lex 4.0K Dec 10 07:42 ../
lrwxrwxrwx 1 lex lex    4 Jan 29 09:53 current -> basic_polybar/
drwxr-xr-x 2 lex lex 4.0K Dec 23 08:06 basic_polybar/
```


If this doesn't do it for you let me know, I want to get you up and running

**Comment by dariusity** [comment]
Ahh now I gotcha. As your instructions seemed like asking to symlink the theme folder to ~/.config/leftwm/themes/current/basic_polybar but I have already figured it out. My first time on a tiling WM, I might have questions too in the future which may not be "issues", do you have forum thread or something that could communicate? I do have a couple of questions regarding popups from browsers tend to not show itself most of the time, and just simple ones on more detailed configurations as for the time being the documentation can't really be helpful. Cheers.

**Comment by dariusity** [comment]
You seem pretty occupied, I tried to recompile for AUR again and after installed it gave error asking me to recompile with RUST_BACKTRACE=1.
